README.md

```diff
@@ -106,6 +106,7 @@ Sample integration with `xlsx-import` are placed in [./samples](./samples) direc
 * [NodeJS sample](./samples/nodejs/) of **importing an invoice** - it is pure JS example which runs on nodejs.
 * [NodeJS + TS sample](./samples/nodejs-ts/) of **importing an invoice** - it is Typescript example that can be transpiled down to pure JS or run directly with ts-node.
 * [ExpressJS sample](./samples/express/) - it is a small service created with ExpressJS can parse xlsx files with concrete structure
+* [React sample](./samples/samples-react/) - it is a web app created with React that displays parsed xlsx file
```

Comment:
please move to `./samples/react/`

Reply:
yeah, I'll fix it, it's just CRA, it wont allow you to create project with this name